docs/contributors/release-process.md

```diff
@@ -56,7 +59,14 @@ git push origin v0.1
 ### After the release
 
 1. Announce the release on Twitter, Discord and other channels.
-2. Deploy the release:
+2. Deploy the release image to [Dockerhub](https://hub.docker.com/layers/statusteam/nim-waku/a5f8b9/images/sha256-88691a8f82bd6a4242fa99053a65b7fc4762b23a2b4e879d0f8b578c798a0e09?context=explore) by triggering [the manual Jenkins deployment job]([https://ci.status.im/job/nim-waku/job/deploy-wakuv2-prod/)](https://ci.infra.status.im/job/nim-waku/job/manual/build).
```

Comment:
Nitpick but looks like doesn't render well.

Reply:
Oh! Thanks. Missed this!

Reply:
Fixed in [9275e62](https://github.com/waku-org/nwaku/pull/1610/commits/9275e62684e49966ba1280bc8acba32c2ed7727f)